runtime/src/iree/base/internal/threading_test.cc

```diff
@@ -155,15 +152,9 @@ TEST(ThreadTest, PriorityOverride) {
   EXPECT_NE(nullptr, override2);
 
   // Wait for the thread to finish.
-  iree_notification_await(
-      &entry_data.barrier,
-      +[](void* entry_arg) -> bool {
-        auto* entry_data = reinterpret_cast<struct entry_data_t*>(entry_arg);
-        return iree_atomic_load_int32(&entry_data->value,
-                                      iree_memory_order_relaxed) == 1;
-      },
-      &entry_data, iree_infinite_timeout());
-  iree_notification_deinitialize(&entry_data.barrier);
+  while (iree_atomic_load_int32(&entry_data.value, iree_memory_order_acquire) !=
```

Comment:
I think we may want `iree_wait_until(iree_timeout_as_deadline_ns(iree_make_timeout_ms(1)))` in here (or `std::this_thread::yield();`, but mixing in C++ threading into a test of non-C++ threading sketches me out). It's not strictly required as eventually the OS will force a context switch but adding spinloops doesn't feel great :)

Reply:
done.